cli/Cargo.toml

```diff
@@ -54,7 +54,11 @@ license = "Apache-2.0"
 
 [[bin]]
 name = "splinter"
-path = "src/main.rs"
+path = "src/splinter/main.rs"
+
+[[bin]]
+name = "scabbard"
+path = "src/scabbard/main.rs"
```

Comment:
I'm not sure how much sense it makes to package both CLI's together; maybe they belong in their own respective crates.